src/e2e/features/1-authentication.feature

```diff
@@ -2,7 +2,7 @@ Feature: Login
 
   Scenario: Bob wants to login to ZAC
     Given "Bob" navigates to "zac" with path "/"
-    When "Bob" logs in with username "testuser1" and password "testuser1"
+    When "Bob" logs in with username "testuser1" and password "TER5yM!cte5V"
```

Comment:
Hmmm, dit willen we niet in onze Git repo lijkt me. Het is open source he, nu heeft de hele wereld deze wachtwoorden gezien. Dit moeten GitHub Action secrets worden lijkt mij.